app/client/src/layout/sideMenu.js

```diff
@@ -51,7 +51,7 @@ const SideMenu = (props) => {
     <div id="content">
       <span className="slide">
         <a href="#" onClick={openSlideMenu}>
-          <img src={Menu} id="Menu" alt="Profile" />
+          <MenuRoundedIcon id="Menu" alt="profile" style={{ color: "white" }} />
```

Comment:
This is replacing the hamburger menu icon? We found when initially testing the old version that it isn't considered good practice to select things based on DOM attributes like _id_ or _alt_. For non-text elements of the site, the recommendation was for some sort of aria attribute, as if it were a screen reader looking at it.

Reply:
@Bottle7

Reply:
I will look into changing this. The reason it is currently like that is because the previous team chose to position this element with the id selector in a stylesheet and I haven't converted everything to material UI yet.